_posts/2020-08-24-changelog-129.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #129"
+date: 2020-08-24 04:01:13 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5745 -->
+* Implement [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/) run configuration.
+  So now you can build your WASM library using `wasm-pack` just from the IDE
```

Comment:
Hm, actually, it's not only about build. Maybe it's better to write something like "So now you can run any `wasm-pack` command just from the IDE"? What do you think?
Sorry for confusing first comment